gst/gstvalue.h

```diff
@@ -26,6 +26,17 @@
 
 G_BEGIN_DECLS
 
+/**
+ * GST_PRINTABLE_CHAR_ARGS:
+ * @c: a char value to output
+ *
+ * To be used with %c when chars may be invalid. Ensures that c is printable by
+ * substituting invalid chars by '?'. This avoids producing invalid strings,
+ * glib warnings and assertions.
+ */
+#define GST_PRINTABLE_CHAR_ARGS(c) \
+        (G_UNLIKELY((c) < ' ' || (c) & 0x80) ? ('?') : ((gchar)(c)))
```

Comment:
ASCII letter 0x7F (<DEL>) doesn't have 0x80 bit set, but it's not a printable character

Reply:
True. That one should not cause problems, but I'll change it to compare with 0x7E instead.